app/data_model/answer_store.py

```diff
@@ -0,0 +1,192 @@
+import re
+
+from collections import OrderedDict
+
+
+class Answer(object):
+    def __init__(self, answer_id, block_id, value, answer_instance=None):
+        answer_instance = answer_instance or 0
+
+        self.answer_id = answer_id
+        self.block_id = block_id
+        self.answer_instance = answer_instance
+        self.value = value
+
+    def equals(self, answer):
```

Comment:
Oh just noticed that it's not really testing equality more that it's the same answer identifier so might not make sense here. Maybe the old `same` name was more reflective of that?